contracts/abstract/ReaperBaseStrategy.sol

```diff
@@ -141,37 +141,6 @@ abstract contract ReaperBaseStrategy is
         return harvestLog.length;
     }
 
-    /**
```

Comment:
contract size issues.

Reply:
I think it would be preferable to keep the log slice function. It could be used by our front-end for an automated solution of our APY display problems. From example you call the function with (5), get [100, 90, 53535, 123, 88, 95] <-- an algorithm can analyze this and throw out the 3rd number, then make a new average.

Reply:
There is already a function to get the size of the log, and any element in the log can be queried directly by index. I have some other ideas to reduce contract size in #11 but that leads to a lot of refactoring then.